client/state/user-suggestions/test/actions.js

```diff
@@ -32,39 +31,36 @@ describe( 'actions', () => {
 	} );
 
 	describe( '#requestUserSuggestions', () => {
-		useNock( ( nock ) => {
+		beforeAll( () => {
 			nock( 'https://public-api.wordpress.com:443' )
 				.get( '/rest/v1.1/users/suggest?site_id=' + siteId )
 				.reply( 200, deepFreeze( sampleSuccessResponse ) );
 		} );
 
-		test( 'should dispatch properly when receiving a valid response', () => {
-			const dispatchSpy = sinon.stub();
-			dispatchSpy.withArgs( sinon.match.instanceOf( Promise ) ).returnsArg( 0 );
+		test( 'should dispatch properly when receiving a valid response', async () => {
+			expect.assertions( 3 );
```

Comment:
This assertion looks like it's not needed? The test has to either run through all three `expect` calls or fail by throwing an exception, nothing else.